src/index.html

```diff
@@ -4,24 +4,407 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
     />
     <title>Catalog</title>
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
```

Comment:
Linking a SCSS file directly in HTML does not work because browsers do not understand SCSS syntax. You should compile SCSS to CSS and link the compiled CSS file instead.